wafer_space/projects/templatetags/precheck_tags.py

```
@@ -0,0 +1,190 @@
+
+def _get_version_icon(*, is_latest: bool | None) -> tuple[str, str]:
+    """Return (icon_name, css_class) for version status."""
+    if is_latest is True:
+        return ("cloud-check-fill", "text-success")
+    if is_latest is False:
+        return ("cloud-arrow-up-fill", "text-warning")
+    return ("cloud", "text-muted")
+
+
+def _get_version_string(
+    check: ManufacturabilityCheck,
```

Ruff (0.14.8)

Unused function argument: `check`

(ARG001)